### DISCUSSION TITLE
Non-modal dialogs? (alerts, subwindows, etc.)

### DISCUSSION THREAD

**Discussion by Frozenlock** [picocss/pico#586] [Help]
It seems that Picocss is treating _any_ dialog element as a modal: adds a blurry background, fixes the position and prevents interaction with other elements.

Is there something that can be done to have normal (non-modal) dialogs?

**Comment by dev-willis** [comment]
Perhaps the modal styles should apply to elements with `role="modal"` instead of dialog elements?

**Reply by ouvreboite** [reply]
That would be nice.
We could also have the role="status" for something like a toast/snackbar

**Comment by iain3000** [comment]
an additional wrinkle is that `open` is not intended to be toggled in the way that pico recommends: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog#open

**Comment by tommy4st** [comment]
The current way of applying `open`, also prevents using the [Popover API](https://developer.mozilla.org/en-US/docs/Web/API/Popover_API/Using).
